### ISSUE TITLE
Cannot recognize  ensei_name_05.png

### ISSUE THREAD

**Issue by kevin01523** [mrmin123/kancolle-auto#11]
←[94m[2015-09-20 12:44:08] Let's send an expedition out!←[0m
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[log] CLICK on L(470,563)@S(0)[0,0 1366x768]
[debug] waiting for ensei_name_05.png to appear
[debug] ensei_name_05.png has not appeared.
[error] script stopped with error in line 299
[error] FindFailed ( can not find ensei_name_05.png on the screen. )
Exception in thread "main" java.lang.IllegalStateException: No match found
        at java.util.regex.Matcher.group(Unknown Source)
        at org.sikuli.scriptrunner.JythonScriptRunner.findErrorSourceWalkTrace(J
ythonScriptRunner.java:326)
        at org.sikuli.scriptrunner.JythonScriptRunner.findErrorSource(JythonScri
ptRunner.java:290)
        at org.sikuli.scriptrunner.JythonScriptRunner.runPython(JythonScriptRunn
er.java:200)
        at org.sikuli.scriptrunner.JythonScriptRunner.runScript(JythonScriptRunn
er.java:162)
        at org.sikuli.basics.SikuliScript.main(SikuliScript.java:181)

i dont know what causes it but it seems to not see or determine exped 5 image it runs fine on my version of the script i even copied the image from it

**Comment by mrmin123** [comment]
@kevin01523 Could you try replacing the image with this one and re-running?: 
![ensei_name_05](https://cloud.githubusercontent.com/assets/4326866/9979402/93b1bd24-5f1f-11e5-9eb1-655200524353.png)
I remember having similar issues before with another expedition, and updating the image seemed to help. I just tried exp 5 with the old image and the script crashed like you reported, but worked with this image.

**Comment by kevin01523** [comment]
ok its working now hmm it uses the OCR right for this function?or image matching?

**Comment by mrmin123** [comment]
No, this is still image matching. OCR is used only for the timers. I did, however, raise the matching threshold require by Sikuli because it was sometimes matching on incorrect portions of the screen. I suspect that's the reason why the images that worked on your fork doesn't work on mine.

**Comment by kevin01523** [comment]
ah maybe thats why sometimes the script stops here and there for unable to find some image but works again after i restart the script hmm maybe you can try out on how on onegai ooyodo script uses self.clickWithRandomOffset(Pattern("worlds.png").targetOffset(-38,6)) target offset you can set it using the ide since i edited my own copy of it on my pc to fix some wrong offsets that causes the script to click the wrong button or world in fight runner. what i mean what is you can make the mouse click a certain point of a huge image that matches screen using target offset well we can try minimizing or reducing the  matching threshold and test if it can run for more than 12 hours

**Comment by mrmin123** [comment]
I ran into the issue most specifically with the damage images (dmg_light, dmg_moderate, dmg_critical) because they're so small, but looking at it now I specifically raised their similarity threshold to 0.95. I think I did the global increase to 0.8 because there were other buttons that were being mis-recognized elsewhere with the default 0.7.

I do see the advantage of lowering the threshold back to 0.7: even if the user doesn't have the same exact expeditions unlocked, there's a higher chance of matching (the presence of a background on the expedition list is why things might not match), but that could be rectified by lowering its specific threshold to 0.7, leaving the global as-is. Then again, I seem to recall sometimes Sikuli choosing the wrong expedition due to lax similarity constraints in the past... Not sure what the correct method is.

**Comment by kevin01523** [comment]
also i noticed sometimes the script would always stop at 3-2-a unable to find line ahead button

**Comment by mrmin123** [comment]
Are you on the latest version of the tool? I changed the Line Ahead button recognition to `exact()` a while back because Sikuli would choose the Line Abreast button instead (the characters are very similar), but haven't had an issue with it on both Windows and Linux...

**Comment by kevin01523** [comment]
yes im on the latest  version  i did clone and updated then to the latest bot windows and linux

**Comment by mrmin123** [comment]
Could you try running it with c0316e62dd and see if that helps at all?

**Comment by kevin01523** [comment]
ok ill boot up my ubuntu then try it out

**Comment by kevin01523** [comment]
```
^Ckevin@ubuntu:~/SikuliX$ ./runScript -d 2 -r/home/kevin/kancolle-auto/kancolle_to.sikuli
running SikuliX-Script: -Xmx512M -Dfile.encoding=UTF-8 -Dsikuli.FromCommandLine 
-jar ./sikuli-script.jar -d 2 -r/home/kevin/kancolle-auto/kancolle_auto.sikuli
Picked up JAVA_TOOL_OPTIONS: -javaagent:/usr/share/java/jayatanaag.jar 
[debug] Running on Java 7 (1.7.0_79-b14)
[debug] ResourceLoaderBasic: check: we are running on arch: i386
[debug] ResourceLoaderBasic: check: using Java at: /usr/lib/jvm/java-7-openjdk-i386/jre/
[debug] Screen: initScreens: basic initialization (1 Screen(s) found)
[2015-09-23 00:44:17] Reading config file
[2015-09-23 00:44:17] Config loaded!
[2015-09-23 00:44:17] Starting kancolle_auto
[2015-09-23 00:44:17] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:44:33] At Home!
[2015-09-23 00:44:33] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:44:39] No, no fleets to receive!
[2015-09-23 00:44:39] Refreshing Home!
[debug] waiting for quests_main.png to appear
[debug] quests_main.png has appeared.
[log] CLICK on L(901,129)@S(0)[0,0 1360x768]
[debug] waiting for top_menu_return.png to appear
[debug] top_menu_return.png has appeared.
[debug] waiting for top_menu_return.png to appear
[debug] top_menu_return.png has appeared.
[log] CLICK on L(398,527)@S(0)[0,0 1360x768]
[debug] waiting for top_menu_return.png to appear
[debug] top_menu_return.png has appeared.
[debug] waiting for top_menu_return.png to appear
[debug] top_menu_return.png has appeared.
[log] CLICK on L(398,527)@S(0)[0,0 1360x768]
[2015-09-23 00:45:05] At Home!
[2015-09-23 00:45:05] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:45:10] No, no fleets to receive!
[2015-09-23 00:45:10] Navigating to Expedition menu!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(539,345)@S(0)[0,0 1360x768]
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[log] CLICK on L(1013,315)@S(0)[0,0 1360x768]
[debug] waiting for expedition_screen_ready.png to appear
[debug] expedition_screen_ready.png has appeared.
[2015-09-23 00:45:19] Let's send an expedition out!
[debug] waiting for ensei_area_05.png to appear
[debug] ensei_area_05.png has appeared.
[debug] waiting for ensei_area_05.png to appear
[debug] ensei_area_05.png has appeared.
[log] CLICK on L(707,513)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_38.png to appear
[debug] ensei_name_38.png has appeared.
[debug] waiting for ensei_name_38.png to appear
[debug] ensei_name_38.png has appeared.
[log] CLICK on L(636,400)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(1017,520)@S(0)[0,0 1360x768]
[2015-09-23 00:45:32] Trying to send out fleet 3 for expedition 38
[debug] waiting for fleet_3.png to appear
[debug] fleet_3.png has appeared.
[debug] waiting for fleet_3.png to appear
[debug] fleet_3.png has appeared.
[log] CLICK on L(761,196)@S(0)[0,0 1360x768]
[2015-09-23 00:45:43] Fleet 3 needs resupply!
[debug] waiting for supply_side.png to appear
[debug] supply_side.png has appeared.
[log] CLICK on L(362,284)@S(0)[0,0 1360x768]
[2015-09-23 00:45:44] Lets resupply fleets!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for fleet_3.png to appear
[debug] fleet_3.png has appeared.
[log] CLICK on L(545,197)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[log] CLICK on L(458,198)@S(0)[0,0 1360x768]
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[log] CLICK on L(1042,518)@S(0)[0,0 1360x768]
[debug] waiting for supply_not_available.png to appear
[debug] supply_not_available.png has appeared.
[2015-09-23 00:46:02] Done resupplying!
[2015-09-23 00:46:02] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:46:11] Going Home!
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(411,335)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[2015-09-23 00:46:15] At Home!
[2015-09-23 00:46:15] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:46:20] No, no fleets to receive!
[2015-09-23 00:46:20] Navigating to Expedition menu!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(539,345)@S(0)[0,0 1360x768]
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[log] CLICK on L(1013,315)@S(0)[0,0 1360x768]
[debug] waiting for expedition_screen_ready.png to appear
[debug] expedition_screen_ready.png has appeared.
[2015-09-23 00:46:27] Let's send an expedition out!
[debug] waiting for ensei_area_05.png to appear
[debug] ensei_area_05.png has appeared.
[debug] waiting for ensei_area_05.png to appear
[debug] ensei_area_05.png has appeared.
[log] CLICK on L(707,513)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_38.png to appear
[debug] ensei_name_38.png has appeared.
[debug] waiting for ensei_name_38.png to appear
[debug] ensei_name_38.png has appeared.
[log] CLICK on L(636,400)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(1017,520)@S(0)[0,0 1360x768]
[2015-09-23 00:46:39] Trying to send out fleet 3 for expedition 38
[debug] waiting for fleet_3.png to appear
[debug] fleet_3.png has appeared.
[debug] waiting for fleet_3.png to appear
[debug] fleet_3.png has appeared.
[log] CLICK on L(761,196)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for ensei_start.png to appear
[debug] ensei_start.png has appeared.
[debug] waiting for ensei_start.png to appear
[debug] ensei_start.png has appeared.
[log] CLICK on L(965,522)@S(0)[0,0 1360x768]
[debug] waiting for exp_started.png to appear
[debug] exp_started.png has appeared.
[2015-09-23 00:46:58] Expedition sent!: Expedition 38 (ETA 2015-09-23 03:41:23)
[2015-09-23 00:46:59] Let's send an expedition out!
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[log] CLICK on L(475,512)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_05.png to appear
[debug] ensei_name_05.png has appeared.
[debug] waiting for ensei_name_05.png to appear
[debug] ensei_name_05.png has appeared.
[log] CLICK on L(638,370)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(1017,520)@S(0)[0,0 1360x768]
[2015-09-23 00:47:11] Trying to send out fleet 2 for expedition 5
[2015-09-23 00:47:17] Fleet 2 needs resupply!
[debug] waiting for supply_side.png to appear
[debug] supply_side.png has appeared.
[log] CLICK on L(362,284)@S(0)[0,0 1360x768]
[2015-09-23 00:47:18] Lets resupply fleets!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for fleet_2.png to appear
[debug] fleet_2.png has appeared.
[log] CLICK on L(516,197)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[log] CLICK on L(458,198)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[log] CLICK on L(458,198)@S(0)[0,0 1360x768]
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[log] CLICK on L(1042,518)@S(0)[0,0 1360x768]
[debug] waiting for supply_not_available.png to appear
[debug] supply_not_available.png has appeared.
[2015-09-23 00:47:48] Done resupplying!
[2015-09-23 00:47:48] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:47:57] Going Home!
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(411,335)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[2015-09-23 00:48:02] At Home!
[2015-09-23 00:48:02] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:48:07] No, no fleets to receive!
[2015-09-23 00:48:07] Navigating to Expedition menu!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(539,345)@S(0)[0,0 1360x768]
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[log] CLICK on L(1013,315)@S(0)[0,0 1360x768]
[debug] waiting for expedition_screen_ready.png to appear
[debug] expedition_screen_ready.png has appeared.
[2015-09-23 00:48:15] Let's send an expedition out!
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[log] CLICK on L(475,512)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_05.png to appear
[debug] ensei_name_05.png has appeared.
[debug] waiting for ensei_name_05.png to appear
[debug] ensei_name_05.png has appeared.
[log] CLICK on L(638,370)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(1017,520)@S(0)[0,0 1360x768]
[2015-09-23 00:48:28] Trying to send out fleet 2 for expedition 5
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for ensei_start.png to appear
[debug] ensei_start.png has appeared.
[debug] waiting for ensei_start.png to appear
[debug] ensei_start.png has appeared.
[log] CLICK on L(965,522)@S(0)[0,0 1360x768]
[debug] waiting for exp_started.png to appear
[debug] exp_started.png has appeared.
[2015-09-23 00:48:45] Expedition sent!: Expedition 5 (ETA 2015-09-23 02:18:10)
[2015-09-23 00:48:46] Let's send an expedition out!
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[log] CLICK on L(475,512)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_02.png to appear
[debug] ensei_name_02.png has appeared.
[debug] waiting for ensei_name_02.png to appear
[debug] ensei_name_02.png has appeared.
[log] CLICK on L(638,276)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(1017,520)@S(0)[0,0 1360x768]
[2015-09-23 00:48:59] Trying to send out fleet 4 for expedition 2
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[log] CLICK on L(792,195)@S(0)[0,0 1360x768]
[2015-09-23 00:49:06] Fleet 4 needs resupply!
[debug] waiting for supply_side.png to appear
[debug] supply_side.png has appeared.
[log] CLICK on L(362,284)@S(0)[0,0 1360x768]
[2015-09-23 00:49:08] Lets resupply fleets!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[log] CLICK on L(576,196)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[debug] waiting for supply_all.png to appear
[debug] supply_all.png has appeared.
[log] CLICK on L(458,198)@S(0)[0,0 1360x768]
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[debug] waiting for supply_available.png to appear
[debug] supply_available.png has appeared.
[log] CLICK on L(1042,518)@S(0)[0,0 1360x768]
[debug] waiting for supply_not_available.png to appear
[debug] supply_not_available.png has appeared.
[2015-09-23 00:49:25] Done resupplying!
[2015-09-23 00:49:25] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:49:34] Going Home!
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(411,335)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[2015-09-23 00:49:37] At Home!
[2015-09-23 00:49:37] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:49:42] No, no fleets to receive!
[2015-09-23 00:49:42] Navigating to Expedition menu!
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(539,345)@S(0)[0,0 1360x768]
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[debug] waiting for expedition.png to appear
[debug] expedition.png has appeared.
[log] CLICK on L(1013,315)@S(0)[0,0 1360x768]
[debug] waiting for expedition_screen_ready.png to appear
[debug] expedition_screen_ready.png has appeared.
[2015-09-23 00:49:49] Let's send an expedition out!
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[debug] waiting for ensei_area_01.png to appear
[debug] ensei_area_01.png has appeared.
[log] CLICK on L(475,512)@S(0)[0,0 1360x768]
[debug] waiting for ensei_name_02.png to appear
[debug] ensei_name_02.png has appeared.
[debug] waiting for ensei_name_02.png to appear
[debug] ensei_name_02.png has appeared.
[log] CLICK on L(638,276)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(1017,520)@S(0)[0,0 1360x768]
[2015-09-23 00:50:01] Trying to send out fleet 4 for expedition 2
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[debug] waiting for fleet_4.png to appear
[debug] fleet_4.png has appeared.
[log] CLICK on L(792,195)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for ensei_start.png to appear
[debug] ensei_start.png has appeared.
[debug] waiting for ensei_start.png to appear
[debug] ensei_start.png has appeared.
[log] CLICK on L(965,522)@S(0)[0,0 1360x768]
[debug] waiting for exp_started.png to appear
[debug] exp_started.png has appeared.
[2015-09-23 00:50:20] Expedition sent!: Expedition 2 (ETA 2015-09-23 01:19:45)
[2015-09-23 00:50:21] Focus on KanColle!
[log] App.focus KanColleTool Viewer(0) #0
[log] switchApp: KanColleTool Viewer
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:50:30] Going Home!
[debug] waiting for home_side.png to appear
[debug] home_side.png has appeared.
[log] CLICK on L(411,335)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[2015-09-23 00:50:33] At Home!
[2015-09-23 00:50:33] Are there returning expeditions to receive?
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:50:38] No, no fleets to receive!
[2015-09-23 00:50:38] Navigating to Sortie menu!
[debug] waiting for sortie.png to appear
[debug] sortie.png has appeared.
[log] CLICK on L(539,345)@S(0)[0,0 1360x768]
[debug] waiting for combat.png to appear
[debug] combat.png has appeared.
[debug] waiting for combat.png to appear
[debug] combat.png has appeared.
[log] CLICK on L(573,315)@S(0)[0,0 1360x768]
[debug] waiting for combat_area_3.png to appear
[debug] combat_area_3.png has appeared.
[debug] waiting for combat_area_3.png to appear
[debug] combat_area_3.png has appeared.
[log] CLICK on L(648,520)@S(0)[0,0 1360x768]
[debug] waiting for combat_panel_3-2.png to appear
[debug] combat_panel_3-2.png has appeared.
[debug] waiting for combat_panel_3-2.png to appear
[debug] combat_panel_3-2.png has appeared.
[log] CLICK on L(935,282)@S(0)[0,0 1360x768]
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[debug] waiting for decision.png to appear
[debug] decision.png has appeared.
[log] CLICK on L(1025,520)@S(0)[0,0 1360x768]
[debug] waiting for senseki_off.png to appear
[debug] senseki_off.png has appeared.
[2015-09-23 00:50:54] Checking fleet condition...
[2015-09-23 00:51:05] Light damage: 0; moderate damage: 0; critical damage: 0
[2015-09-23 00:51:08] Starting sortie!
[debug] waiting for combat_start.png to appear
[debug] combat_start.png has appeared.
[debug] waiting for combat_start.png to appear
[debug] combat_start.png has appeared.
[log] CLICK on L(970,521)@S(0)[0,0 1360x768]
[debug] waiting for compass.png to appear
[debug] compass.png has appeared.
[debug] waiting for compass.png to appear
[debug] compass.png has appeared.
[log] CLICK on L(676,322)@S(0)[0,0 1360x768]
[debug] waiting for P(formation_line_ahead.png) S: 0.9 to appear
[debug] P(formation_line_ahead.png) S: 0.9 has not appeared.
[error] script stopped with error in line 302
[error] FindFailed ( can not find P(formation_line_ahead.png) S: 0.9 on the screen. )
[error] --- Traceback --- error source first
line: module ( function ) statement 
38: util (  wait_and_click )     kc_window.wait(pic, time)
66: combat (  go_sortie )     wait_and_click(self.kc_window, Pattern('formation_line_ahead.png').similar(0.90), 10)
291: main (  init )     combat_item.go_sortie()
[error] --- Traceback --- end --------------
kevin@ubuntu:~/SikuliX$ 
```

ok it stopped again

**Comment by mrmin123** [comment]
Is the formation menu showing up in 10 seconds after the compass is
pressed? That, or take a screen capture of the Line Ahead button and
replace formation_line_ahead.png? Those are the two alternatives I can
think of... (again, hesitant about lowering the pattern matching too much
because of the similarity to the line abreast button)

**Comment by kevin01523** [comment]
i guess increasing the waiting time would do the trick or use something like this http://imgur.com/RjvR7TZ then set targetOffset on the button to click

**Comment by mrmin123** [comment]
Here's a commit with the wait times increased: 96748a4cb52d16125079e48316ffb495693baa82
I guess the larger image + targetOffset is a possible method, but I'm hesitant to use it because of the shipgirl image showing through... Could you let me know how the increased timer works if you get the chance?

**Comment by kevin01523** [comment]
ok it does work now but it clicked line abreast consecutively

**Comment by mrmin123** [comment]
I had that issue previously when the pattern matching wasn't exact. Try this (longer wait times, exact formation image matching): 2dfa6c1

**Comment by kevin01523** [comment]
[debug] Screen: initScreens: basic initialization (1 Screen(s) found)
[error] script stopped with error in line 4
[error] SyntaxError ( !!WHILE IMPORTING!! mismatched input ')' expecting NEWLINE (combat.py, line 66) )

**Comment by mrmin123** [comment]
Typo in the line. That's what I get for pushing without testing. Fixed in 97783fa2fa4282e36fc8844e97ab53288775163a
